fetch.bs

```diff
@@ -1693,11 +1696,11 @@ and "<code>webidentity</code>" as fetches with those destinations skip service w
 "<code>xslt</code>" as that too can cause script execution. It is not included in the list as it is
 not always relevant and might require different behavior.
 
-<div class=note>
- <p>The following table illustrates the relationship between a <a for=/>request</a>'s
- <a for=request>initiator</a>, <a for=request>destination</a>, CSP directives, and features. It is
- not exhaustive with respect to features. Features need to have the relevant values defined in their
- respective standards.
+<div class=note id="destination-table">
+ <p>The following table llustrates the relationship between a
```

Comment:
This entire paragraph's diff seems unnecessary, and mostly just inserts a typo.

Reply:
Fixed